src/main/kotlin/com/autonomousapps/tasks/ComputeAdviceTask.kt

```diff
@@ -310,6 +401,27 @@ internal class DependencyAdviceBuilder(
         StandardTransform(coordinates, declarations, supportedSourceSets, buildPath, isKaptApplied).reduce(usages)
       }
   }
+
+  /**
+   * Returns whether this is a KMP non-common target that can just defer to a parent common declaration in
+   * [kmpCommonDeclarations].
+   *
+   * @see GradleVariantIdentification.kmpAttribute
```

Comment:
My IDE cannot resolve this. Looks like it should be `Coordinates.kmpAttribute`.

Reply:
[9580098](https://github.com/autonomousapps/dependency-analysis-android-gradle-plugin/pull/919/commits/958009802ed7360f0ef74239fd32c9d0b45bc603)